test/runTests.cpp

```diff
@@ -112,7 +114,7 @@ bool runRandomTests(int numTests, EdlibAlignMode mode, bool findAlignment) {
         // printf("\n");
 
         start = clock();
-        EdlibAlignResult result = edlibAlign(
+        EdlibAlignResult result = edlibAlign<char>(
```

Comment:
There are more calls of edlibAlign in these file, what about those, how is it that those work?